beginner_source/blitz/data_parallel_tutorial.py

```diff
@@ -0,0 +1,254 @@
+"""
+Optional: Data Parallelism
+============
```

Comment:
I see ` WARNING: Title underline too short.` during sphinx build. Can you fix all such warnings?